IDEAS/Buildings/Components/Interfaces/RectangularZoneTemplateInterface.mo

```diff
@@ -205,6 +205,10 @@ partial model RectangularZoneTemplateInterface
   parameter SI.TemperatureDifference dTiAvg=2
     "Amplitude of variation of monthly average indoor temperature"
     annotation(Dialog(tab="Floor", group="Slab on ground", enable=(bouTypFlo == IDEAS.Buildings.Components.Interfaces.BoundaryType.SlabOnGround)));
+    parameter Modelica.Units.SI.Temperature T_start_gro[3]={TeAvg,TeAvg,TeAvg}
```

Comment:
Maybe also make this T_start_gro[3]=fill(TeAvg, 3)? Such that it is in correspondence with SlabOnGround.mo. Creating the same nGropLay parameter would be too much i think.

Reply:
Ok! I have changed this. Adding nLayGro is currently not possible, because it is explicitly defined in the SlabOnGround model. In SlabOnGround, nLayGro is derived from the MultiLayer model in which nLay=3.